pages/index.tsx

```diff
@@ -2,11 +2,18 @@ import { GetStaticProps } from 'next';
 import { handleImages } from 'src/utils/imageHandler';
 import { Employee, massageEmployee } from 'src/employees';
 import { getEmployeesUrl } from 'src/utils/api/getEmployees';
+import { CaseJSON } from '../src/case/Case';
+import { CaseList } from '../src/case/cases';
 
 export { default } from 'src/index';
 
+function shuffle(array: Array<CaseJSON>) {
+  array.sort(() => Math.random() - 0.5);
```

Comment:
Sort mutates, but it also returns. To ensure the original won't be "ruined" you can copy it before. It's not uncommon to see something like 

```
  [...array].sort(() => Math.random() - 0.5);
```

Reply:
Yes, going over it now and chaging :D